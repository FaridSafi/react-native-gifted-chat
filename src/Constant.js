import { Platform } from 'react-native';
export const MIN_COMPOSER_HEIGHT = Platform.select({
    ios: 33,
    android: 41,
});
export const MAX_COMPOSER_HEIGHT = 200;
export const DEFAULT_PLACEHOLDER = 'Type a message...';
export const DATE_FORMAT = 'll';
export const TIME_FORMAT = 'LT';
//# sourceMappingURL=Constant.js.map