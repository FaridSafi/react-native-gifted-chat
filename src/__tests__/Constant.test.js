import * as Constant from '../Constant';

it('should compare Constant with snapshot', () => {
  expect(Constant).toMatchSnapshot();
});
