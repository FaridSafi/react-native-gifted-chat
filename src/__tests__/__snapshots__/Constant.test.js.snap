// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should compare Constant with snapshot 1`] = `
Object {
  "DATE_FORMAT": "ll",
  "DEFAULT_PLACEHOLDER": "Type a message...",
  "MAX_COMPOSER_HEIGHT": 200,
  "MIN_COMPOSER_HEIGHT": 33,
  "TIME_FORMAT": "LT",
}
`;
